2023-07-14 10:06:49,164 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
2023-07-14 10:07:24,202 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
2023-07-14 10:19:37,793 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
2023-07-14 10:19:41,763 ERROR: Exception on /edit_profile [GET] [in C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\app\routes.py", line 70, in edit_profile
    form = EditProfileForm()
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\wtforms\form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: EditProfileForm.__init__() missing 1 required positional argument: 'original_username'
2023-07-14 10:19:45,243 ERROR: Exception on /edit_profile [GET] [in C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\app\routes.py", line 70, in edit_profile
    form = EditProfileForm()
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\wtforms\form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: EditProfileForm.__init__() missing 1 required positional argument: 'original_username'
2023-07-14 10:19:56,010 ERROR: Exception on /edit_profile [GET] [in C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\app\routes.py", line 70, in edit_profile
    form = EditProfileForm()
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\Documents\backend\microblog\venv\Lib\site-packages\wtforms\form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: EditProfileForm.__init__() missing 1 required positional argument: 'original_username'
2023-07-14 10:20:40,675 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
2023-07-14 10:38:48,172 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
2023-07-14 10:40:15,652 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
2023-07-14 10:40:50,821 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
2023-07-14 10:41:10,956 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
2023-07-14 10:41:45,982 INFO: Microblog startup [in C:\Users\admin\Documents\backend\microblog\app\__init__.py:28]
